plugins/blip/basics/chartbasicsfactory.js

```diff
@@ -70,7 +70,7 @@ var BasicsChart = React.createClass({
   componentWillMount: function() {
     var basicsData = this.props.patientData.basicsData;
     if (basicsData.sections == null) {
-      basicsData = _.assign(basicsData, basicsState);
+      basicsData = _.assign({}, basicsData, _.cloneDeep(basicsState));
```

Comment:
Yes, good point. However, this will definitely fall under our stylistic principle of "stick with ES5 in ES5 modules." In the uploader redux stuff though, I _am_ using `Object.assign` and if you see lodash out of habit there, definitely call me out on it!